fvm.go

```diff
@@ -0,0 +1,54 @@
+package ffi
+
+import (
+	"github.com/filecoin-project/ffi-stub/cgo"
+	"github.com/filecoin-project/go-state-types/abi"
+	"github.com/filecoin-project/go-state-types/network"
+	"github.com/ipfs/go-cid"
+)
+
+type FVM struct{}
+
+type FVMOpts struct {
+	FVMVersion uint64
+	Externs    cgo.Externs
+
+	Epoch          abi.ChainEpoch
+	BaseFee        abi.TokenAmount
+	BaseCircSupply abi.TokenAmount
+	NetworkVersion network.Version
+	StateBase      cid.Cid
+	Manifest       cid.Cid
+	Tracing        bool
+}
+
+type ApplyRet struct {
```

Comment:
reason for extra fields?
https://github.com/filecoin-project/filecoin-ffi/blob/bb281b5dfcfb7f485056604c570a3c92f103cea9/fvm.go#L152

Reply:
These are present in master https://github.com/filecoin-project/filecoin-ffi/blob/9fd89f165adc097e3d1e0223d14aeae573f03e81/fvm.go#L162